content/blog/2018-07-05-meanings-effect-size.Rmd

```diff
@@ -0,0 +1,162 @@
+---
+title: Can we call mean differences "effect sizes"?
+author: Pierre Dragicevic
+date: '2018-07-05'
+slug: meanings-effect-size
+categories: []
+tags: [effect size,guidelines]
+bibliography: 2018-07-05-meanings-effect-size.bib
+link-citations: true
+---
+
+The current draft of the [transparent statistics guidelines](https://transparentstats.github.io/guidelines/) has a FAQ and exemplar on effect sizes. During a [meeting](http://transparentstatistics.org/chi2018/) we ran at [CHI](https://chi2018.acm.org/) to collect feedback, a participant strongly objected to our use of the term "effect size" in the guidelines. This prompted me to investigate further to make sure we haven't missed anything in deciding the terminology. Here is what I found.
+
+# We should report effect sizes. But what are effect sizes?
```

Comment:
Per #2, since article title is implicitly h1 (`# ...`), these headings should be h2 (`## ...`)